opt/cs50/bin/flask

```diff
@@ -39,7 +57,7 @@ if [[ "$1" == "run" ]]; then
     # Spawn flask
     FLASK_APP="$FLASK_APP" \
     FLASK_DEBUG="${FLASK_DEBUG:-0}" \
-    FLASK_ENV="development" \
+    FLASK_ENV="$FLASK_ENV" \
```

Comment:
These 3 lines are still not needed?

Reply:
So without that, the vars don't seem to be available to the process that gets invoked?

Reply:
The variables that we give default values in the script (just `FLASK_ENV` atm) can just be exported and it'll be available to the flask subprocess (and any other subprocesses of this script, if we were to start more) or we can set it before the command, but I don't think we need the other two.

Reply:
Actually we give a default value to `FLASK_APP` too if `application.py` is found. So we can just prefix these two lines with `export` and we won't have to set these before the command.

Reply:
Well, was trying to keep the variables scoped to our wrapper, since there doesn't seem to be an upside in exposing them to the shell?

Which don't we need precisely?

Reply:
The exported variables will be scoped to the wrapper and its subprocesses if we export them in the wrapper. They won't be exported in the shell.

Reply:
Ah yes, was thinking `profile.d` still. Changed.